test_data/series7_device_config.yaml

```diff
@@ -1,9 +1,28 @@
 # Which BEL names are global buffers for nextpnr?
-global_buffers:
+global_buffer_bels:
 - BUFG
 - BUFGCTRL
 - VCC
 - GND
+# Which cell names are global buffers, and which pins should use dedicated routing resources
+global_buffer_cells:
+  - cell: BUFG
+    pins: # list of pins that use global resources
+     - name: I # pin name
+       guide_placement: true # attempt to place so that this pin can use dedicated resources
+       max_hops: 10 # max hops of interconnect to search
+     - name: O
+       force_routing: true # the net connected to this pin _must_ use dedicated routing only
```

Comment:
Not sure if there are cases in which this might happen, but, is that possible that some nets may need to pass through non-dedicated routing resources?

I think this might be handled in pnr though, where a pin that has the force_routing attribute tries as much as it can to b e routed using dedicated routing, and falls back to use general interconnect only if the first failed.

On a side note, I think this attribute might be renamed to sth like `force_dedicated_routing` or similar.

Reply:
There is enough short-hop/tile-local general routing available to the global router for this purpose already that this case _should_ be OK, its only the longer distance general routing that it will avoid. I'd rather keep this as an error for now, and we can revisit if that's a problem.

Reply:
I agree about the attribute name though, and have made that change as well as adding a note about max_hops.